lib/health-data-standards.rb

```diff
@@ -34,56 +33,56 @@
 require_relative 'health-data-standards/import/provider_import_utils'
 require_relative 'health-data-standards/import/hdata/metadata_importer'
 
-require_relative 'health-data-standards/models/thing_with_codes'
-require_relative 'health-data-standards/models/result_value'
-require_relative 'health-data-standards/models/coded_result_value'
-require_relative 'health-data-standards/models/physical_quantity_result_value'
-require_relative 'health-data-standards/models/cda_identifier'
-require_relative 'health-data-standards/models/reference'
+# require_relative 'health-data-standards/models/thing_with_codes'
+# require_relative 'health-data-standards/models/result_value'
+# require_relative 'health-data-standards/models/coded_result_value'
+# require_relative 'health-data-standards/models/physical_quantity_result_value'
+# require_relative 'health-data-standards/models/cda_identifier'
+# require_relative 'health-data-standards/models/reference'
 require_relative 'health-data-standards/models/entry'
 require_relative 'health-data-standards/models/allergy'
```

Comment:
we use this `allergy `? and `concept` on line 71

Reply:
these were added due to circle failures in ecqm-api or web

allergy was needed because it inherits from entry which is embedded_in record. once record was required, so was entry and allergy.

concept was added when value_set became required.  value_set embeds_many concepts and wouldn't work without concept also used.